.circleci/config.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - run:
           name: use opensource tool client9/misspell to correct commonly misspelled English words
           command: |
+            go get -u github.com/client9/misspell/cmd/misspell
```

Comment:
This is a bug in the latest pouchlinter image because I forget to retrieve this binary after removing gometalinter. Should we update  pouchlinter image first? @allencloud

Reply:
Yes, I have updated that. so we can remove this line of code.